### PR TITLE
Fix only-allow-pnpm example

### PR DESCRIPTION
**Pull request by Jolg42**
It looks like the "`" back tick makes yarn and pnpm hang.

Removing them make yarn console the message and pnpm works like expected!

### CODE REVIEW

File: docs/only-allow-pnpm.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ To prevent devs from using other package managers, add the following `preinstall
 ```json
 {
   "scripts": {
-    "preinstall": "node -e \"!process.env.npm_config_user_agent.startsWith('pnpm/')&&!console.log('Use `npx pnpm install` to install dependencies in this repository\\n')&&process.exit(1)\""
```
Comment (zkochan) [inline docs/only-allow-pnpm.md]: I think it would be better to still quote it. Maybe through `\\'`

Comment (Jolg42) [reply docs/only-allow-pnpm.md]: I agree, doing it now :)

File: docs/only-allow-pnpm.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ To prevent devs from using other package managers, add the following `preinstall
 ```json
 {
   "scripts": {
-    "preinstall": "node -e \"!process.env.npm_config_user_agent.startsWith('pnpm/')&&!console.log('Use npx pnpm install to install dependencies in this repository\\n')&&process.exit(1)\""
+    "preinstall": "node -e \"!process.env.npm_config_user_agent.startsWith('pnpm/') && !console.log('Use \\`npx pnpm install\\` to install dependencies in this repository\\n')&&process.exit(1)\""
```
Comment (zkochan) [inline docs/only-allow-pnpm.md]: I thought the ` had issues so you'll use the single quote

Comment (Jolg42) [reply docs/only-allow-pnpm.md]: So yes indeed but it looks like the double escape works with the ` after testing it :)

**Comment by Jolg42** [comment]
I just added 2 spaces to make it more readable but this version works for me perfectly! Thank you for the example by the way, because I messed up the dependencies with yarn once and didn't want that to happen again 😅

**Pull request merged**